### PR TITLE
Remove CurrentVersion and BuildInfo

### PR DESCRIPTION
**Pull request by slozier**

### CODE REVIEW

**Comment by slide** [comment]
Can you also remove `Util/MSBuildCommunityTasks` as part of this since it is not needed anymore?

**Comment by slide** [comment]
Looks like the Implementation.cs changes broke something. The previous sys.version is: `2.7.9 (IronPython 2.7.9 (2.7.9.0) on Mono 4.0.30319.42000 (64-bit))`

The new sys.version is: `2.7.9 (2.7.9 (2.7.9.0) on Mono 4.0.30319.42000 (64-bit))`

This is causing an issue in most of the tests

**Comment by slozier** [comment]
I can double-check but I think MSBuildCommunityTasks is needed for packaging?

**Comment by slide** [comment]
I wonder if that can be replaced

File: Build/Versioning.targets [hunk 1]
```diff
@@ -1,93 +0,0 @@
...
-    <ShortReleaseLevel Condition="'$(ReleaseLevel)' == 'alpha'">a</ShortReleaseLevel>
-    <ShortReleaseLevel Condition="'$(ReleaseLevel)' == 'beta'">b</ShortReleaseLevel>
-    <ShortReleaseLevel Condition="'$(ReleaseLevel)' == 'candidate'">rc</ShortReleaseLevel>
-    <ShortReleaseLevel Condition="'$(ReleaseLevel)' == 'final'">f</ShortReleaseLevel>
-
-    <DisplayVersion Condition="'$(ReleaseLevel)' == 'final' and '$(MicroVersion)' == '0'">$(MajorVersion).$(MinorVersion)</DisplayVersion>
-    <DisplayVersion Condition="'$(ReleaseLevel)' == 'final' and '$(MicroVersion)' != '0'">$(MajorVersion).$(MinorVersion).$(MicroVersion)</DisplayVersion>
-    <DisplayVersion Condition="'$(ReleaseLevel)' != 'final'">$(MajorVersion).$(MinorVersion).$(MicroVersion)$(ShortReleaseLevel)$(ReleaseSerial)</DisplayVersion>
-
-    <PackageVersion Condition="'$(ReleaseLevel)' == 'final'">$(MajorVersion).$(MinorVersion).$(MicroVersion)</PackageVersion>
```
Comment (slide) [inline Build/Versioning.targets]: We still need PackageVersion

**Pull request merged**